docs/style.css

```diff
@@ -616,3 +616,126 @@ blockquote {
 .animation-list {
   text-align: left;
 }
+
+/*-----------------------------------*\
+  $DARK-MODE
+\*-----------------------------------*/
+
+.night-light-toggle-button .night-light-checkbox {
```

Comment:
please avoid descended selector, it complicates your CSS and couple HTML to CSS too much.